configs/neox_arguments.md

```diff
@@ -843,6 +843,29 @@ Model Arguments
 
 
 
+- **dim_att**: int
```

Comment:
we should either have unified args (across mamba, rwkv, transformers) for these, or prepend these args with whatever block type they're targeting (e.g. `rwkv_dim_att`).